app/controllers/alerts_list_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Controller for showing Alert records
 class AlertsListController < ApplicationController
+  include AlertsHelper
```

Comment:
>For instance: If the controller name is AlertsListController, AlertsListHelper is automatically included for you. 
In this particular case when the helper name is not ```AlertsListHelper```, it still works if you don't include it in the controller (i.e you are able to access the helper method in the alerts_list view) .